GitCommands/Git/GitCommandCache.cs

```diff
@@ -94,10 +94,8 @@ public static void Add(string cmd, byte[] output, byte[] error)
         public static void CleanCache()
         {
             lock (queue)
-            {
-                if (queue != null)
-                    queue.Clear();
-            }
+                queue?.Clear();
```

Comment:
I don't want to lose `{ }`

In fact, I'd love to see them everywhere :sunglasses:

Reply:
@RussKie i kinda agree.  Sometimes terser doesnt make it easier to read it.